00 - Contribute to the Obsidian Hub/03 Contributor Notes/03.03 Scripts and Automation/Updating MOC files.md

```diff
@@ -0,0 +1,12 @@
+The script `update-mocs.py` examines the current Hub vault contents and updates the MOC files (🗂️*.md) to match.
+
+## Running the script
+
+To run it:
+
+```bash
+cd obsidian-hub/.github/scripts
+./update-mocs.py
```

Comment:
```suggestion
python3 ./update-mocs.py
```
Like above it should include python3 so that Windows users can run it without issues.